<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditForm</class>
 <widget class="QDialog" name="EditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Contact</string>
  </property>
  <widget class="QLabel" name="lname">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButtonEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fnameInputEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Address">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>57</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="fname">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnameInputEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="number">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>57</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Number:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButtonEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="addrInputEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>331</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nickname">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>57</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
  </widget>
  <widget class="QLabel" name="type">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nicknameInputEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeBoxEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberInputEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>501</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
